synapse/lib/stormtypes.py

```diff
@@ -7112,6 +7270,47 @@ async def _methGetFormcount(self):
         todo = s_common.todo('getFormCounts')
         return await self.viewDynCall(todo, ('view', 'read'))
 
+    @stormfunc(readonly=True)
+    async def _methGetPropCount(self, propname, valu=undef):
+        propname = await tostr(propname)
+
+        if valu is not undef:
```

Comment:
```
        if valu is undef:
            valu = s_common.novalu
        else:
            valu = await toprim(valu)
```

Allows us to prevent leaking the undef sentinel outside of storm code, and then we can rely on the `s_common.novalu` sentinel in the view.py code.

Reply:
Updated to this